packages/core/src/App/Containers/Modals/information-submitted-modal/information-submitted-modal.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Dialog } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import { useStore, observer } from '@deriv/stores';
+
+const InformationSubmittedModal = observer(() => {
+    const { ui } = useStore();
+    const { is_kyc_information_submitted_modal_open: is_open, toggleKycInformationSubmittedModal } = ui;
+
+    return (
+        <Dialog
+            className='information-submitted-modal'
+            title={localize('Information updated')}
```

Comment:
Can we use Localize component for `title` prop?

Reply:
`title` is of string type in this component, so used `localize` function